Makefile

```diff
@@ -24,9 +24,8 @@ node_modules: package.json
 .PHONY: gen
 gen: node_modules
 	curl https://meyerweb.com/eric/tools/css/reset/reset.css | \
-		python gencss.py | \
-		node_modules/.bin/prettier --stdin --stdin-filepath=x.scss > \
```

Comment:
I'd prefer that we keep `prettier` (and the stripping of trailing newlines) in this target. While we do enforce that `pre-commit` tests are passing, it's nicer if our scripts don't emit files that will (on the first run) fail our hooks.